datafusion/doc/src/lib.rs

```diff
@@ -63,6 +63,88 @@ impl Documentation {
     ) -> DocumentationBuilder {
         DocumentationBuilder::new(doc_section, description, syntax_example)
     }
+
+    /// Output the `Documentation` struct in form of custom Rust documentation attributes
+    pub fn to_doc_attribute(&self) -> String {
```

Comment:
Maybe it would also help helpful here to note this is to help migration to doc comment attributes